babble/src/main/java/io/mosaicnetworks/babble/service/BabbleServiceBinder.java

```diff
@@ -73,7 +73,7 @@ protected void doBindService() {
         if (getActivity().bindService(new Intent(getActivity(), BabbleService2.class), mConnection, Context.BIND_AUTO_CREATE)) {
             mShouldUnbind = true;
         } else {
-            Log.e("MY_APP_TAG", "Error: The requested service doesn't " +
+            Log.e("ERROR", "Error: The requested service doesn't " +
```

Comment:
The tag here should not be "ERROR". I have been using Class name - BabbleServiceBinder in this case. It offers some clue as to where the error is thrown.

Reply:
Based on this answer https://stackoverflow.com/questions/8355632/how-do-you-usually-tag-log-entries-android
Shall we add:
private static final String TAG = BabbleServiceBinder.class.getName();

Reply:
Fair enough. I am happy with hardcoding too. Anything but "ERROR" will do.